ynput/ayon-core
# Load and manage products from a library project

## Changelog Description
This PR is to support loading assets from the library project while the users can update and set versions with the containers from the loaded assets in the scene inventory tools. In the DCC hosts, you need to add the `project_name` as part of the container data, otherwise it will use the current project name instead(which results to the Entity N/A shown in the #983)
Resolve https://github.com/ynput/ayon-core/issues/983

## Additional info
Make sure using the self.filepath_from_context(context) for the updating function in the loaders.

## Testing notes:
1. Load Assets from the current project and library projects
2. Go to scene inventory to set version or update
3. Should be working.
